packages/create-starter/src/index.ts

```diff
@@ -44,35 +47,39 @@ export async function main() {
       name: 'kit',
       message: 'Which starter kit would you like to use?',
       choices: starters,
-      suggest: (input, choices) => Promise.resolve(choices.filter(c => c.title.includes(input))),
+      suggest: (input, choices) => Promise.resolve(choices.filter((c) => c.title.includes(input))),
```

Comment:
Once the merge conflict is resolved, then it will show the update for this filter function